CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Deprecations
 1. Deprecated ``http_access`` in **aviatrix_controller_config**. This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release.
+2. Deprecated ``bandwidth`` in **WAN/LAN/MGMT interfaces**. This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release.
```

Comment:
Should we also add deprecation notes in each resource doc?
Please see terraform-provider-aviatrix/docs/resources/

Reply:
Added changes to the resource doc.